src/outputs/restart.hpp

```diff
@@ -128,7 +123,7 @@ class RestartReader {
     hid_t filespace = H5Dget_space(dataset);
     int rank = H5Sget_simple_extent_ndims(filespace);
     std::vector<hsize_t> dims(rank);
-    status = H5Sget_simple_extent_dims(filespace, dims.data(), NULL);
+    H5Sget_simple_extent_dims(filespace, dims.data(), NULL);
```

Comment:
Should this (and below) also get a check?
```suggestion
    PARTHENON_HDF5_CHECK((H5Sget_simple_extent_dims(filespace, dims.data(), NULL));
```